ste/folderCreationTracker.go

```diff
@@ -9,6 +9,78 @@ import (
 	"github.com/Azure/azure-storage-azcopy/v10/common"
 )
 
+type TrieNode struct {
+	children map[rune]*TrieNode
+	value    *uint32
```

Comment:
Do we require to store value address or can we simply store the value of that?

Reply:
you are correct we should store the value directly.